src/forms-opportunities/oppty-handlers/low-form-views-handler.js

```diff
@@ -0,0 +1,96 @@
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { FORM_OPPORTUNITY_TYPES } from '../constants.js';
+import { filterForms, generateOpptyData } from '../utils.js';
+
+/**
+ * @param auditUrl - The URL of the audit
+ * @param auditData - The audit data containing the audit result and additional details.
+ * @param context - The context object containing the data access and logger objects.
+ */
+// eslint-disable-next-line max-len
+export default async function createLowFormViewsOpportunities(auditUrl, auditDataObject, scrapedData, context) {
+  const { dataAccess, log } = context;
+  const { Opportunity } = dataAccess;
+
+  // eslint-disable-next-line no-param-reassign
+  const auditData = JSON.parse(JSON.stringify(auditDataObject));
+  log.info(`Syncing high page views low form views opportunity for ${auditData.siteId}`);
+  let opportunities;
+
+  try {
+    opportunities = await Opportunity.allBySiteIdAndStatus(auditData.siteId, 'NEW');
+  } catch (e) {
+    log.error(`Fetching opportunities for siteId ${auditData.siteId} failed with error: ${e.message}`);
+    throw new Error(`Failed to fetch opportunities for siteId ${auditData.siteId}: ${e.message}`);
+  }
+
+  const { formVitals } = auditData.auditResult;
+  // eslint-disable-next-line max-len
+  const formOpportunities = await generateOpptyData(formVitals, context, [FORM_OPPORTUNITY_TYPES.LOW_FORM_VIEWS]);
+  log.debug(`forms opportunities high-page-views-low-form-views: ${JSON.stringify(formOpportunities, null, 2)}`);
+
+  const filteredOpportunities = filterForms(formOpportunities, scrapedData, log);
+  log.info(`filtered opportunities: high-page-views-low-form-views:  ${JSON.stringify(filteredOpportunities, null, 2)}`);
+
+  try {
+    for (const opptyData of filteredOpportunities) {
+      let highPageViewsLowFormViewsOptty = opportunities.find(
+        (oppty) => oppty.getType() === FORM_OPPORTUNITY_TYPES.LOW_FORM_VIEWS
+          && oppty.getData().form === opptyData.form,
+      );
+
+      const opportunityData = {
+        siteId: auditData.siteId,
+        auditId: auditData.auditId,
+        runbook: 'https://adobe.sharepoint.com/:w:/s/AEM_Forms/EeYKNa4HQkRAleWXjC5YZbMBMhveB08F1yTTUQSrP97Eow?e=cZdsnA',
+        type: FORM_OPPORTUNITY_TYPES.LOW_FORM_VIEWS,
+        origin: 'AUTOMATION',
+        title: 'Form page has high views but the form has low views',
+        description: 'The form page has a lot of views but the form has low views due to form being below the fold',
+        tags: ['Forms Conversion'],
+        data: {
+          ...opptyData,
+        },
+        guidance: {
+          recommendations: [
+            {
+              insight: `The Form in the page: ${opptyData.form} is either placed below the fold or is shown as a pop-up/modal`,
```

Comment:
The form in the page: ${opptyData.form} has low discoverability and only ${formViews/pageViews*100}% visitors landing on the page are viewing the form.